scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -357,6 +357,12 @@ impl DefaultForProfile for TomlProfile {
     }
 }
 
+#[derive(Debug, Default, Deserialize, Serialize, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct TomlToolScarbMetadata {
+    pub allow_prebuilt_macros: Option<Vec<String>>,
```

Comment:
Nit:

Proc macros use `[cairo_plugin]` target in Scarb manifest, so just for consistency it might be better to name this
```suggestion
    pub allow_prebuilt_plugins: Option<Vec<String>>,
```